db/db_impl.cc

```diff
@@ -1249,27 +1248,28 @@ std::vector<Status> DBImpl::MultiGet(
          has_unpersisted_data_.load(std::memory_order_relaxed));
     bool done = false;
     if (!skip_memtable) {
-      if (super_version->mem->Get(lkey, value, &s, &merge_context,
-                                  &range_del_agg, read_options)) {
+      if (super_version->mem->Get(lkey, pinnable_val->GetSelf(), &s,
+                                  &merge_context, &range_del_agg,
+                                  read_options)) {
         done = true;
+        pinnable_val->PinSelf();
```

Comment:
nit: may be it would be simpler if we get a refernece to GetSelf to value and pass value to ::Get. And then we call PinSelf once after the two branches.

Reply:
sure, shall I update `GetImpl` similarly? since this piece of code is duplicated from GetImpl